src/Events/Controllers/StorageController.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Threading.Tasks;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services.Interfaces;
+using AutoMapper;
+
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Swashbuckle.AspNetCore.Annotations;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Provides operations for saving and retrieving cloud events from persistent storage.
+    /// </summary>
+    [Authorize]
+    [Route("events/api/v1/storage/events")]
+    public class StorageController : ControllerBase
+    {
+        private readonly IInboundService _inboundService;
+        private readonly ILogger _logger;
+        private readonly IMapper _mapper;
```

Comment:
Not sure if you need a mapper here? This controller should only accept and process objects of the cloudEvent model